### PR TITLE
Create ohai plugin for looker

### PR DESCRIPTION
**Pull request by JamesAwesome**
* Add dependency on ohai cookbook
* Create template for looker ohai plugin
* Add looker::_ohai_plugin to install plugin
* Correct escaping of environment vars in .kitchen.yml
* Bump version

### CODE REVIEW

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/1/
Test FAILed.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/2/
Test FAILed.

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/3/
Test FAILed.

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/4/
Test FAILed.

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/6/
Test FAILed.

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/7/
Test FAILed.

**Comment by JamesAwesome** [comment]
TEST!

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/8/
Test PASSed.

**Comment by JamesAwesome** [comment]
Failures due to issue with docker container & UTF-8 local, see https://github.com/berkshelf/berkshelf-api/issues/112 for details.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/9/
Test FAILed.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/10/
Test FAILed.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/11/
Test PASSed.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/12/
Test PASSed.

File: .kitchen.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ suites:
       - 'looker'
     attributes:
       looker:
-        startup_script_url: ENV['LOOKER_STARTUP_SCRIPT_URL']
-        jar_file_url: ENV['LOOKER_JAR_URL']
+        startup_script_url: <%= ENV['LOOKER_STARTUP_SCRIPT_URL'] %>
```
Comment (davidsayers) [inline .kitchen.yml]: Why ENV variables and not the actual URL?

Comment (JamesAwesome) [inline .kitchen.yml]: Looker doesn't make these variables known to the public

Comment (davidsayers) [inline .kitchen.yml]: Ok, might be worth putting into the README about how to get them.

Rest of it LGTM

Comment (JamesAwesome) [inline .kitchen.yml]: Sounds like a good Idea, I'll clarify

**Comment by davidsayers** [comment]
:+1:

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/13/
Test PASSed.

**Comment by roboshopkeep** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins-devops.posrip.com//job/cookbook-looker/14/
Test PASSed.

**Pull request merged**